samples/algorithms/PhaseFlipCode.qs

```diff
@@ -140,14 +140,14 @@ namespace Sample {
         // parity measurements.
         let indexOfError =
             if (parity01, parity12) == (One, Zero) {
-                0
-            } elif (parity01, parity12) == (One, One) {
-                1
-            } elif (parity01, parity12) == (Zero, One) {
-                2
-            } else {
+            0
```

Comment:
Looks like another case where this line and the following aren't indented correctly. Looks from this diff like the same indent level as the 'if' statement that it's within a block of.

Reply:
Indentation logic will be addressed by a future PR.